core/src/consensus/stake/mod.rs

```diff
@@ -226,6 +231,33 @@ fn revoke(state: &mut TopLevelState, fee_payer: &Address, delegatee: &Address, q
     Ok(())
 }
 
+fn redelegate(
+    state: &mut TopLevelState,
+    fee_payer: &Address,
+    prev_delegatee: &Address,
+    next_delegatee: &Address,
+    quantity: u64,
+) -> StateResult<()> {
+    let delegator = StakeAccount::load_from_state(state, fee_payer)?;
```

Comment:
Check `next_delegatee` is valid candidate like `delegate`
https://github.com/CodeChain-io/codechain/pull/1742/files#diff-a25f9a1ca8af0d023966594384adb0f4R194

Reply:
Thanks. I missed it.